Resources/public/css/theme/views/subitemlist.css

```diff
@@ -92,3 +92,69 @@
     left: 50%;
     position: absolute;
 }
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after {
+    content: "\e606";
+    background: #E6E6E6;
+    color: #737373;
+    font-weight: bold;
+    border: 1px solid #737373;
+}
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-error-priority-cell  .ez-subitem-priority-icons:after {
+    content: "!";
+    background: #BF3E33;
+    color: #fff;
+    font-weight: bold;
+    border: 1px solid #BF3E33;
+}
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-error-priority-cell {
+    background: #FEECEA !important;
+    box-shadow: none;
+    border: 1px double #BF3E33;
+}
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-selected-priority-cell {
+    box-shadow: none;
+    border: 1px double #737373;
+}
+
+.ez-view-subitemlistview .ez-subitem-priority-input,
+.ez-view-subitemlistview .ez-subitem-priority-input:focus,
+.ez-view-subitemlistview .ez-subitem-priority-input.ez-subitem-error-input:focus {
+    border: 0 !important;
+    background-color: transparent !important;
+    box-shadow: none !important;
+}
+
+.ez-view-subitemlistview .ez-subitem-priority-input[readonly],
+.ez-view-subitemlistview .ez-subitem-priority-input[readonly]:focus {
+    background-color: transparent;
+    color: #000000;
+    outline: none;
+}
+
+.ez-view-subitemlistview .ez-subitem-priority-cancel:before {
```

Comment:
please apply https://github.com/ezsystems/PlatformUIBundle/pull/527#issuecomment-195302786, basically, the buttons should get 2 classes:

* for the validate button: `ez-subitem-priority-button` and `ez-subitem-priority-button-validate`
* for the cancel button: `ez-subitem-priority-button` and `ez-subitem-priority-button-validate`

```css
.ez-view-subitemlistview .ez-subitem-priority-button:before {
    font-family: "ez-platformui-icomoon";
}

.ez-view-subitemlistview .ez-subitem-priority-button-cancel:before {
    content: "\E600";
    color: #E46474;
}

.ez-view-subitemlistview .ez-subitem-priority-button-validate:before {
    content: "\EA10";
    color: #1EDCB1;
}

.ez-view-subitemlistview .ez-subitem-priority-button {
    border: 1px solid #737373;
    background: #E6E6E6;
}

.ez-view-subitemlistview .ez-subitem-priority-button-cancel {
    border-left: 0;
}
```

and of course the same organization should be in the other stylesheet.